### ISSUE TITLE
Broken OctetString conversion

### ISSUE THREAD

**Issue by soulraven** [node-red/node-red-nodes#976]
This convertion toString broke the OctetString when MAC as a value is present, because must be converted to HEX.
This problem is not present in SNMP Walk, because is returning the value un processed and make the user to deal with conversion

The value is not consistent between different SNMP nodes and force the user to use different methods to get the same value right.

https://github.com/node-red/node-red-nodes/blob/553ae44ccb55b3dd87c44d514e4068142831af4e/io/snmp/snmp.js#L257

**Comment by dceejay** [comment]
I agree the handling is inconsistent so am willing to make his change. This will be a major version bump as it will be a breaking change for some users.
